### ISSUE TITLE
Naming

### ISSUE THREAD

**Issue by Flyingmana** [OpenMage/organizational#1]
This will be a long running project, and choosing a Name is hard.

Why do we need a new Name?
OpenMage is quite near to the original name, and the License of Magento does not allow to use its trademarks.
I and probably also the rest of the team/contributors dont have the money to pay a Lawyer to defend our usage. Also its not nice to bring the Company behind Magento in such a difficult situation where they could be forced to defend their trademark.

* We will not change existing class Names
* Usage of "Magento" in documentation, Blog, ... will not be allowed

As said, choosing a Name is hard. Having over a Year to do this is much better then less then a week in case of legal complications.

To keep some kind of insider connection I would expect the new Name which has either a connection to a color (Magento => Magenta) ((Magenta is funnily enough trademarked from a different company...)
or a character of a fictional universe (Magento => Magneto) (( yeah I know, this is clearly a trademark minefield)
Also I would suggest, it should keep the "Open" Prefix.

Now discuss, meanwhile I will take suggestions for popcorn sources and using the show to write popcorn reviews. (this is a joke related to http://knowyourmeme.com/memes/popcorn-gifs , but also I want to try out home made popcorn)

**Comment by birchestx** [comment]
OpenMango. RealOpenMango :)

**Comment by tzyganu** [comment]
OpenFork.

**Comment by gplanchat** [comment]
Magneto

**Comment by kkoepke** [comment]
- **Contin** (like continuous or continuation)
**MOne** or **Mone** (like Mage One. MOne maybe sounds a bit too much like Motel One, the hotel most people use on german Magento conferences?)
- play with any synonyms or similarities for mage? (witch, sorcerer, ..., **Open Witch**? 💃 ) 
- what about **Enchant** (like enchanter)?
- **juggle** or Magento plus juggle = **Muggle**? ;-) **Muggle Commerce**? (Harry Potter for sure isnt my thing but isnt it an sorcerer there?)

**Comment by kkoepke** [comment]
**Manco** or **OpenManco** (Manco was an Inca King)
**Maggot** :-)

Or playing around with some Magenta color shades?
https://en.wikipedia.org/wiki/Shades_of_magenta
http://www.workwithcolor.com/magenta-color-hue-range-01.htm

**Comment by colinodell** [comment]
Ideas starting with `Ma`:

 - Magicart
 - MangoCart

Ideas based on the color magenta:

 - Plum Commerce
 - OpenPlum
 - Vialet (see what I did there?)

**Comment by LeeSaferite** [comment]
Unless someone can show me a trademark for Mage by Magento, I honestly think that keeping OpenMage makes the most sense simply because it uses the code namespace and makes it easily clear that it is a fork. We can, and should, reach out to the Magento team to discuss the usage of the name. Again, I see no reason to change as I've never seen 'Mage' as a trademark claimed by Magento.

**Comment by leebolding** [comment]
Exactly - great Lee's think alike ;)

Currently talking with Daniel about this

**Comment by Flyingmana** [comment]
Let me describe my thinking here.

Even when there is no direct case for them to sue this project, there is a risk they still  try/do.
Even when they may not win, it will be expensive and hurt everyone in the ecosystem.

So it is about risk avoidance. Most legal advice has to do with risk avoidance, thats why we have disclaimers everywhere. And even they are sometimes useless, if the law suite has a bit of substance.

So even if there is absolutely no reason, I would still go with a name change just to avoid any possible risk.

But additionally I see risk here, even if Mage is not registered as a trademark itself.
Even if we would register OpenMage as a trademark successfully, it would not lead to 100% protection as [this case](http://www.telegraph.co.uk/technology/facebook/7967323/Facebook-sues-Teachbook-over-name-similarity.html) shows.

Some may say, that there are other companies in the ecosystem who do not had a problem with having "mage" as part of their name. But they are mostly not only enhance the ecosystem, they also enhance the value of Magento Inc itself.
But OpenMage would have a position as a competitor, even if we dont want to actually take share away of M2, it will happen, it probably already happens. And you could say, maintaining a similarity by name is giving us an advantage here. And guess, thats hell of a good reason to sue.

Then there is the point of Magento Inc being a for-profit company, therefore some of its decisions are a bit limited. For example they are required to protect shareholder value. So we will have some kind of domino effect, or chain reaction.(people say Iam not good with metaphors, so ignore them if they are not understandable)
As already stated, OpenMage is probably taking a small share of Magento 2 away, this reduces the shareholder value. Now there are in the end only two options for Magento Inc. Protecting and therefore sue us for everything they can. Or do nothing and risk to get sued by shareholders.
Thats a shitty situation for us, and also for Magento Inc. Because even if non of this happens, its just a day away from possibly being able to happen. (see https://en.wiktionary.org/wiki/sword_of_Damocles )
And I dont want to push Magento Inc into such a situation.

This is a complete theoretical scenario, and I have absolutely no experience in Law besides a few things I did read. But I do like to reduce any risks possible.

**Comment by rlieser** [comment]
I vote "LegacyCodeEnto"!

**Comment by ADDISON74** [comment]
**Magento For Life**, for short MFL. 

Heritage to the original name is a must have for letring people know what is going one around here.

**Comment by sreichel** [comment]
I'm with @LeeSaferite  ... just **OpenMage** and/or **magento-LTS** to show people what this project is about.

**Comment by jacquesbh** [comment]
@sreichel `magento-LTS` is not possible if you've read well.

But I agree with `OpenMage`.

@Flyingmana We should talk with Magento. They can give a document which allows us to use the `OpenMage` name, only to prevent further law suits.
Because as @LeeSaferite said, do they have registration on `Mage`?

And because we don't remove the Magento's license/copyright, we are not taking the project appart. It's Open Source, and they choose that way of mind the first day they decided to use an Open Source license.

**Comment by Flyingmana** [comment]
Update: it seems Magento did change the naming scheme for the Community Edition to "Magento Open Source"

I would assume, this is not helpful for keeping the Name OpenMage

https://magento.com/products/community-edition

**Comment by leesbian** [comment]
support for M1 has also been extended to 2020... so maybe this is all a moot point now?

**Comment by Flyingmana** [comment]
oh, the 2020 is new for me, I only did know about the dynamic notice time of 18 months in advance without a certain date.

If security fixes are good enough for everyone, then yes. But I believe this project is for most users more about additional bugfixes in generel.
But thats another [topic](https://github.com/OpenMage/organizational/issues/4).

Here we talk about the name, and now that Magento also took the word "Open" into its branding for the community edition, we are only a "-ento" away of their brand.

**Comment by leesbian** [comment]
Yeah, I guess the "mission statement" of the project is another issue - when I was actively working on Mage+, part of the mission was to increase the velocity of bugfixes being released - which I don't expect is part of the plans for M1 support (it seems to be security fixes only)

I gave up actively working on Mage+ after my efforts seemed to spur Magento into doing much of this themselves (more frequent CE releases, properly crediting contributors, actively accepting pull requests)

**Comment by Flyingmana** [comment]
Any update regarding the Naming?
Did someone try to work on #3 ?

**Comment by digitalpianism** [comment]
OhMaGento (in three words ^^)

**Comment by inluxc** [comment]
Something latin like:
https://translate.google.com/#en/la/commerce

**commercium** - commerce, trade, intercourse, dealing, communication, barter
**mercatura** - trade, merchandise, trading, commerce, buying, traffic
**conmercium** - trade, intercourse, commerce, barter, commercial intercourse, traffic
**mercatus** -market, marketplace, commerce, traffic, fair, trading

**Comment by ProcessEight** [comment]
How about Magenot?

Because it's Magento...but not.

The more you think about it the more it works on many levels.

**Comment by jonashrem** [comment]
OpenGray (contains "Open", the colour Grey and a reference to Dr. Jean Grey alias Phoenix)

**Comment by sreichel** [comment]
@Flyingmana Since we don't know how Magento sees it, I just asked them to contact me. If or as soon as I get an answer I will let you know.

Maybe we're talking about a problem that doesn't exist. :)

**Comment by Flyingmana** [comment]
@sreichel thx for reminding me, I updated #3

**Comment by kkrieger85** [comment]
Which parts of "OpenMage" are copyrighted by Adobe? Is "mage" too "magento" to be used? 


Personally, I like @inluxc ideas on something latin. 
I'd prefer something with "commerce" in it - so developer/merchants see on first sight, what die OpenSource Software is made for.
